src/LiquidityPoolConverter.ts

```diff
@@ -206,11 +206,14 @@ class IConversionEvent {
 
 function handleConversion(event: IConversionEvent): void {
   const toToken = Token.load(event.toToken.toHexString())
-  const fromAmount = decimal.fromBigInt(event.fromAmount, DEFAULT_DECIMALS)
-  const toAmount = decimal.fromBigInt(event.toAmount, DEFAULT_DECIMALS)
-  const conversionFee = decimal.fromBigInt(event.conversionFee, DEFAULT_DECIMALS)
-  const protocolFee = decimal.fromBigInt(event.protocolFee, DEFAULT_DECIMALS)
+  const fromToken = Token.load(event.fromToken.toHexString())
+  const toDecimals = toToken !== null ? toToken.decimals : DEFAULT_DECIMALS
+  const fromAmount = decimal.fromBigInt(event.fromAmount, fromToken !== null ? fromToken.decimals : DEFAULT_DECIMALS)
```

Comment:
let's make the logic consistent between toDecimals and fromDecimals
in which cases is fromToken === null?
also shouldn't this be fromToken != null and not fromToken !== null?

Reply:
it's never null, but that code won't compile without the nullability check

Reply:
In the newer version of assemblyscript, nullability should be checked by triple equals: https://thegraph.com/docs/en/release-notes/assemblyscript-migration-guide/

In the version we use now it doesn't matter, but this will make it easier to upgrade

Reply:
I think for objects this nullability check is invalid with our current version

Reply:
We're using both in the subgraph already, and both seem to be working (a triple equals that definitely works here: https://github.com/DistributedCollective/Sovryn-subgraph/blob/b6d1b76f073382ba10323d8d121a5536b304d4fa/src/utils/Candlesticks.ts#L91)